src/plugins/documentation/index.ts

```diff
@@ -85,3 +84,9 @@ export async function convertWorkspace(
 
   await helpers.fs.writeFile('docs.json', JSON.stringify(workspace, null, '  '))
 }
+
+type ExpectedOptions = {}
+export default {
+  format: 'documentation',
+  convertWorkspace,
+} as Plugin<ExpectedOptions, ConvertedWorkspace | void>
```

Comment:
It's very minor but I think it would be easier to understand this with a type annotation instead of a type cast, like:

```ts
const plugin: Plugin<ExpectedOptions, ConvertedWorkspace | void> = {
  format: 'documentation',
  convertWorkspace, 
}

export default plugin
```

(is that equivalent?) Since casting has several different purposes, I have to think a bit harder about what the cast is actually doing, vs. a type annotation